static/carousel/js/index.js

```diff
@@ -35,7 +35,7 @@ if (not(window.jQuery) || not(window.Carousel) || not(window.Utils)) {
     (function libsReady() {
         setTimeout(function(){
             if (not(window.jQuery) || not(window.Carousel) || not(window.Utils)) {
-                (++libsCheck < 50) ? libsReady() : console.log('limit reached');
+                (++libsCheck < 50) ? libsReady() : console.warn('limit reached');
```

Comment:
That's the one scrutinizer accepts, it moans with others.
We can remove it or at least add a condition to only use it in debug mode.
Please let me know how to proceed.